transcripts/share-apis/project-maintainers/list-roles.json

```diff
@@ -0,0 +1,48 @@
+{
+  "body": {
+    "active": true,
+    "role_assignments": [
+      {
+        "roles": [
+          "project_viewer"
+        ],
+        "subject": {
+          "id": "U-<UUID>",
+          "kind": "user"
+        }
+      },
+      {
+        "roles": [
+          "project_contributor"
```

Comment:
I wonder if this should be `project_collaborator` or `project_maintainer`, given the use of "contributor" to mean external contribution (via "contributor branches")